pywt/tests/test_matlab_compatibility.py

```diff
@@ -13,11 +13,12 @@
 
 _has_matlab = False
 try:
-    from mlabwrap import mlab
+    from pymatbridge import Matlab
+    mlab = Matlab()
 except ImportError:
     print("To run Matlab compatibility tests you need to have MathWorks "
-          "MATLAB, MathWorks Wavelet Toolbox and mlabwrap Python extension "
-          "installed.")
+          "MATLAB, MathWorks Wavelet Toolbox and the pymatbridge Python "
+          "package installed.")
     _has_matlab = True
 
```

Comment:
Can you rename `_has_matlab` to `_not_has_matlab` or similar? It's pretty confusing as is!